infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -342,7 +332,6 @@ public void shutdown() {
         compactor.cancel(true);
         scheduler.shutdownNow();
         batchWriter.close();
-        executor.shutdownNow();
```

Comment:
A question, wouldn't it be needed to do a super.shutdown() as AbstractServer now has a sharedPool?

Reply:
the sharedPool uses daemon threads. Also, AbstractServer.shutdown only sets a bool.

Reply:
This bool in the AbstractServer is used to check if this server component is shutdown or not and decline all incoming message requests in CorfuMsgHandler returning ERROR_SHUTDOWN_EXCEPTION.

Reply:
hmm, I miss read @StrikingThirteen 's comment, I thought you were asking about shutting down the shared pool executor. Good catch. Done.